launch/twice_uvc_ros_driver.launch

```diff
@@ -0,0 +1,38 @@
+<launch>
```

Comment:
maybe add a small comment about the purpose of this launch file compared to "uvc_ros_driver.launch"?